### DISCUSSION TITLE
I want to remove the experimental IRC support

### DISCUSSION THREAD

**Discussion by pajlada** [Chatterino/chatterino2#5498] [Ideas] [answered]
hello, I want to remove the experimental IRC support.
it's been in an experimental stage, and mostly unsupported for a long time, adding a good amount of complexity to how to we handle things. some things that are 99.9% Twitch channels have to be abstracted to support the 0.01% Irc channels.

got any objections of this? let me know what you use it for

Reasons to keep it that I've gotten so far:

1. For use as an alternative to other IRC clients like irssi or mIRC because it's convenient to have both Twitch chat and IRC chats in the same window
2. As a "roundabout" way of supporting other platforms (e.g. through IRC bridges)
3. Because the code/abstractions we have for it might be useful in the future for other platforms

**Comment by Felanbird** [comment]
ok 👍

**Comment by TETYYS** [comment]
i use it for IRC, its convenient to have chat things in one window

**Reply by pajlada** [reply]
same

**Comment by Mm2PL** [comment]
Should we ever want to add another platform all these abstractions will be needed. I don't think removing IRC to get rid of these abstractions is a good move. Although I don't use IRC a lot, I do use it. We've suggested an IRC bridge for youtube livestream chat.

**Comment by 8thony** [comment, marked as answer]
- Done in #5547
